batch_convert.qml

```diff
@@ -451,6 +459,26 @@ MuseScore {
                 id: traverseSubdirs
                 text: qsTr("Process\nSubdirectories")
             } // traverseSubdirs
+            Row {
+                spacing: 5
+                CheckBox {
+                    id: filterContent
+                    text: qsTr("Filter files with")
+                    }
+                TextField {
+                    id: contentFilterString
+                    text: ""
+                    enabled: filterContent.checked
+                    placeholderText: "E.g. *Concerto*"
+                    width: 200
+                }
+                CheckBox {
+                    id: filterWithRegExp
+                    text: qsTr("Use Regular Expression")
+                    enabled: filterContent.checked
+                    checked: true
```

Comment:
Can we have this checkbox under the `contentFilterString` (and hence indented)? Would save screen width and sort that possible issue with a longer translation.

Reply:
![image](https://user-images.githubusercontent.com/50494/187638550-881b7e50-a853-47fe-bc73-817fa8081cac.png)

Reply:
Yes, please!